features/admin.applications.v1/components/wizard/minimal-application-create-wizard.tsx

```diff
@@ -1132,20 +1136,29 @@ export const MinimalAppCreateWizard: FunctionComponent<MinimalApplicationCreateW
                         && isFAPIAppCreationEnabled
                         && (
                             <div className="pt-0 mt-0">
-                                <Field
-                                    data-componentid={ `${ testId }-fapi-app-checkbox` }
-                                    name={ "isFAPIApp" }
-                                    required={ false }
-                                    type="checkbox"
-                                    value={ [ "isFAPIApp" ] }
-                                    children={ [
-                                        {
-                                            label: t("applications:forms.generalDetails" +
-                                                ".fields.isFapiApp.label" ),
-                                            value: "fapiApp"
-                                        }
-                                    ] }
-                                />
+                                <div style={ { alignItems: "center", display: "flex" } }>
```

Comment:
Let's move the inline styles to scss file

Reply:
fixed with [3a0b2fe](https://github.com/wso2/identity-apps/pull/6097/commits/3a0b2fee82c33f4d8e81dec05d773c557332182c)